### PR TITLE
Added CreatedDate Lable for en

### PR DESCRIPTION
**Pull request by ORabpornpra**

### CODE REVIEW

**Comment by ORabpornpra** [comment]
@stonelasley I leave this for you to take a look.

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -54,7 +54,8 @@
       "REPORT_DURATION": "Default Duration (days)",
       "RECEIPT": "Receipts",
       "RECEIPT_IMAGE_QUALITY": "Image Quality",
-      "RECEIPT_SAVE_GALLERY": "Save to Gallery"
+      "RECEIPT_SAVE_GALLERY": "Save to Gallery",
+      "CreatedDate": "Created Date"
```
Comment (stonelasley) [inline src/assets/i18n/en.json]: Change the name to EXPENSE_CREATED_DATE

**Comment by stonelasley** [comment]
@ORabpornpra this looks good but change the name to EXPENSE_CREATED_DATE in the en.json file and then on the actual page you need to add the translate pipe. Look at this page for an example of how to use the translate pipe. 
https://github.com/UintaGroup/axpense/blob/master/src/pages/report-list/report-list.html

**Review by stonelasley — COMMENTED** [review]
@ORabpornpra this looks good but change the name to EXPENSE_CREATED_DATE in the en.json file and then on the actual page you need to add the translate pipe. Look at this page for an example of how to use the translate pipe.
https://github.com/UintaGroup/axpense/blob/master/src/pages/report-list/report-list.htmlent

**Comment by stonelasley** [comment]
@ORabpornpra declining this PR and approving your other.

**Pull request closed without merging**